origin-discovery/apollo/index.js

```diff
@@ -1,5 +1,8 @@
+require('dotenv').config()
+require('envkey')
```

Comment:
should this be in a try/catch statement so that when running in local dev environment we setting ENVKEY is optional ? like same as what you did on event-listener: https://github.com/OriginProtocol/origin/blob/master/origin-discovery/listener/listener.js#L2